recipes/libharu/bld.bat

```diff
@@ -0,0 +1,16 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
```

Comment:
Can you use `ninja` or `jom` to make use of parallelisation?

Reply:
Thanks very much for the suggestion, I have switched to `ninja`, as you recommend.